components/Typeahead.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  AsyncTypeahead,
+  AsyncTypeaheadProps,
+  Menu,
+  MenuItem,
+  TypeaheadModel,
+} from 'react-bootstrap-typeahead'; // Typeahead
+import 'react-bootstrap-typeahead/css/Typeahead.css'; //Typeahead styling
+import Helmet from 'react-helmet'; // Helmet
+import 'bootstrap/dist/css/bootstrap.min.css'; //Bootstrap styling
+import { Box, Center, Divider, VStack, Text } from '@chakra-ui/layout'; // Chakra UI
+
+// Typeahead props
+type Props<T extends TypeaheadModel> = Pick<
+  AsyncTypeaheadProps<T>,
+  'isLoading' | 'onSearch' | 'renderMenuItemChildren'
+> & {
+  results: Array<T>;
+  placeholder: string;
+};
+
+export default function Typeahead<T extends TypeaheadModel>(props: Required<Props<T>>) {
+  const { isLoading, onSearch, renderMenuItemChildren, results, placeholder } = props;
+  const filterBy = () => true;
```

Comment:
I think we want to make this a custom prop? What is the function of this `filterBy` function?